pallets/pools-system/src/migrations/mod.rs

```diff
@@ -154,19 +180,33 @@ pub mod altair {
 		pub token_price: BalanceRatio,
 	}
 
-	#[frame_support::storage_alias]
-	type Epoch<T: Config> = StorageDoubleMap<
-		Pallet<T>,
+	pub struct EpochPrefix;
+	impl StorageInstance for EpochPrefix {
+		const STORAGE_PREFIX: &'static str = "Epoch";
+
+		fn pallet_prefix() -> &'static str {
+			"Pools"
+		}
+	}
+	type Epoch<T> = StorageDoubleMap<
+		EpochPrefix,
 		Blake2_128Concat,
 		<T as Config>::TrancheId,
 		Blake2_128Concat,
 		<T as Config>::EpochId,
 		OldEpochDetails<<T as Config>::Rate>,
 	>;
 
-	#[frame_support::storage_alias]
-	pub type Order<T: Config> = StorageDoubleMap<
-		Pallet<T>,
+	pub struct OrderPrefix;
+	impl StorageInstance for OrderPrefix {
+		const STORAGE_PREFIX: &'static str = "Order";
+
+		fn pallet_prefix() -> &'static str {
+			"Pools"
```

Comment:
Maybe make `"Pools"` a constant that's returned by all of these `pallet_prefix` functions.